ha/test/unit/cluster_validation/all-combinations.json

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
all-combinations.json 
e are having separate json for each then why again combined file needed

Reply:
assuming it for unit test only